tests/test_client.py

```diff
@@ -337,77 +324,70 @@ def test_get_fills_specified_success(self):
-                pairs=PAIRS,
+                market=MARKETS,
                 limit=limit,
                 startingBefore=startingBefore
             )
             assert result == json_obj
 
-    # ------------ get_my_trades ------------
+    # # ------------ get_my_trades ------------
 
     def test_get_my_trades_no_pairs_error(self):
         client = Client(PRIVATE_KEY_1)
         with pytest.raises(TypeError) as error:
             client.get_my_trades()
-        assert 'required positional argument: \'pairs\'' in str(error.value)
+        assert 'required positional argument: \'market\'' in str(error.value)
 
     def test_get_my_trades_default_success(self):
         client = Client(PRIVATE_KEY_1)
         with requests_mock.mock() as rm:
             json_obj = tests.test_json.mock_get_trades_json
-            uri = 'https://api.dydx.exchange/v1/dex/trades' \
-                + '?makerAccountOwner=' + client.public_address \
-                + '&makerAccountNumber=' + str(client.account_number) \
-                + '&pairs=' + ','.join(PAIRS)
+            uri = 'https://api.dydx.exchange/v2/trades' \
+                + '?accountOwner=' + client.public_address \
+                + '&accountNumber=' + str(client.account_number) \
+                + '&market=' + ','.join(MARKETS)
             rm.get(uri, json=json_obj)
             result = client.get_my_trades(
-                pairs=PAIRS
+                market=MARKETS,
             )
             assert result == json_obj
 
     def test_get_my_trades_specified_success(self):
         client = Client(PRIVATE_KEY_1)
-        limit = 1234
+        limit = 99
         startingBefore = datetime.datetime.utcnow().isoformat()
         with requests_mock.mock() as rm:
             json_obj = tests.test_json.mock_get_trades_json
-            uri = 'https://api.dydx.exchange/v1/dex/trades' \
-                + '?makerAccountOwner=' + client.public_address \
-                + '&makerAccountNumber=' + str(client.account_number) \
-                + '&pairs=' + ','.join(PAIRS) \
+            uri = 'https://api.dydx.exchange/v2/trades' \
+                + '?accountOwner=' + client.public_address \
+                + '&accountNumber=' + str(client.account_number) \
+                + '&market=' + ','.join(MARKETS) \
                 + '&limit=' + str(limit) \
                 + '&startingBefore=' + startingBefore
             rm.get(uri, json=json_obj)
             result = client.get_my_trades(
-                pairs=PAIRS,
+                market=MARKETS,
                 limit=limit,
                 startingBefore=startingBefore
             )
             assert result == json_obj
 
-    # ------------ get_trades ------------
-
-    def test_get_trades_no_pairs_error(self):
-        client = Client(PRIVATE_KEY_1)
-        with pytest.raises(TypeError) as error:
-            client.get_trades()
-        assert 'required positional argument: \'pairs\'' in str(error.value)
```

Comment:
should replace these tests with a test that does not specify `markets`

Reply:
with the `get_trades`, it will not raise an error if a `market` is not specified, only if it is not specified on the `get_my_trades`, this is true for `fills`, `orders`, and `trades`. not sure if we are trying to keep that same pattern.

Reply:
ok that's fine I guess, but the tests should ensure that it's okay to leave markets blank and that the correct url gets called

Reply:
the tests ensure that the `get_my_(trades,fills,orders)` fails without a market and succeeds with one. i can add another test.